app/components/searchworks4/collection_info_component.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Searchworks4
+  class CollectionInfoComponent < ViewComponent::Base
+    delegate :get_collection, to: :helpers
+    def collection
+      get_collection
```

Comment:
This isn't memoized and it's already called once to set the title. It would be preferable to pass `@parent` in to the initializer from the view context.